akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerSpec.scala

```diff
@@ -1174,27 +1174,27 @@ class HttpServerSpec extends AkkaSpec(
       netIn.sendComplete()
     })
 
-    "support request length verification" which afterWord("is defined via") {
-
-      class LengthVerificationTest(maxContentLength: Int) extends TestSetup(maxContentLength) {
-        val entityBase = "0123456789ABCD"
-        def sendStrictRequestWithLength(bytes: Int) =
-          send(s"""POST /foo HTTP/1.1
+    def supportRequestLengthVerification(verboseErrorMessages: Boolean = true) =
```

Comment:
Most of the changes here are because of the added indentation. What I actually added was running the test both with `verboseErrorMessage = true` and `false` and adapting the expected error messages accordingly.